components/src/dynamo/profiler/utils/config_modifiers/vllm.py

```
@@ -163,6 +186,75 @@ def convert_config(
+            return args
+
+        prefix_caching_enabled = (
+            "--enable-prefix-caching" in args
+            and "--no-enable-prefix-caching" not in args
+        )
+        if mamba_cache_mode != "align" and not prefix_caching_enabled:
+            return args
+
+        try:
+            mamba_align_floor = get_mamba_cache_align_block_size(model_name_or_path)
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)